values/dev/network/calibrationapi-archive.yaml

```diff
@@ -6,7 +6,7 @@ nodeSelector:
 IPFS:
   enabled: false
 
-Lotus:
+lotus:
```

Comment:
Are you sure we need to have this change?
I think we need to revert or make an additional changes